runtimes/getting-started.mdx

```diff
@@ -95,8 +95,8 @@ Check out the runtime subpages for steps on how to get started!
   - [canvas](https://www.npmjs.com/package/@rive-app/canvas)
   - [canvas-lite](https://www.npmjs.com/package/@rive-app/canvas-lite)
   - [canvas (advanced)](https://www.npmjs.com/package/@rive-app/canvas-advanced)
-  - [webgl](https://www.npmjs.com/package/@rive-app/webgl)
-  - [webgl (advanced)](https://www.npmjs.com/package/@rive-app/webgl-advanced)
+  - [webgl](https://www.npmjs.com/package/@rive-app/webgl2)
```

Comment:
I kept the names of the links "webgl" because this is really the only webgl package we're talking about moving forward. I thought that adding "2" would cause confusion.